validphys2/src/validphys/process_options.py

```diff
@@ -256,6 +257,20 @@ def _dybosonpt_xq2map(kin_dict):
     return x, ET2
 
 
+def _dymll_xq2map(kin_info):
```

Comment:
Here, I'm not sure how I should do this without having sqrts as a variable, so I have just kept it as a kinematic variable in the yaml file

Reply:
sqrts is always one of the variables, as it comes from the name of the file.

Reply:
Ok, so can I leave this _dymll_xq2map function the same and just remove sqrts from the kinematics yaml file? Or will this function (and/or script) need editing too?

Reply:
I don't think I can access the reports on that comment, it looks like I need an nnpdf username and password

Reply:
Do you have access to the wiki?

Reply:
Yes, that login doesn't seem to be working for me though for this report

Reply:
[The infor](https://www.wiki.ed.ac.uk/display/nnpdfwiki/HTTP+Secrets) <--- you need the password for the vp server

Reply:
Perfect, thank you

Reply:
Here is the report showing the comparison of the old vs new data: [https://vp.nnpdf.science/8EHbLXgpTTOQWrbIiQMXhg==/](https://vp.nnpdf.science/8EHbLXgpTTOQWrbIiQMXhg==/)

They look identical to me but I'm not sure how I might be able to compare the actual data values.